build.sbt

```diff
@@ -109,6 +111,8 @@ lazy val chiselFrontend = (project in file("chiselFrontend")).
   ).
   dependsOn(coreMacros)
 
+// Hack to avoid circular dependency on chisel.
+val root = Project("chisel", file("."))
 
 lazy val chisel = (project in file(".")).
```

Comment:
I am no sbt expert either, but it looks like we now have `chisel` and `root` as the same project. Is root a magical val that projects using chisel will set and thus override or overrule this assignment thus unmanageBase in root will still work?